src/assets/ba_data/python/bauiv1/_hooks.py

```diff
@@ -125,7 +125,13 @@ def root_ui_level_icon_press() -> None:
 
 def root_ui_menu_button_press() -> None:
     from bauiv1._appsubsystem import UIV1AppSubsystem
+    from babase import app
+
+    if app.classic is None:
+        logging.exception('Classic not present.')
+        return
 
+    app.classic.resume()
```

Comment:
Is there a bug that this is addressing? In general I'm trying to untangle classic gameplay stuff from ui stuff so if resume needs to be called somewhere it should probably go under classic somewhere.

Reply:
So the bug is, when you are in game and you open the in game menu, it pauses the game, but instead of pressing resume to resume the game, try to press the  top right button which opens the in game button again, it will close the in game menu but doesn't resume the game.

So in the previous fix, I added the __del__ function to resume whenever the in game menu is deleted, but it turns out it created another problem. So now pressing the top right button when the in game menu is open closes the in game menu and resumes it, but if the in game menu is open and you press the settings button, it quickly opens and closes the settings window because the __del__ function runs and the game resumes, so the window is closed.

I thought instead of doing something to the in game menu window, I could just go to the button and resume it when it's pressed so no problems will occur. If you got any other ideas to fix it, it will be nice.

Reply:
Thanks for the explanation. So I removed the resume() call from the __del__() call and added one at the callback for the menu button in classic._appmode._root_ui_menu_press(). So I *think* it should cover all cases now. Let me know if you find any case still broken...